sotopia/database/logs.py

```diff
@@ -14,29 +19,21 @@ class EpisodeLog(JsonModel):
 
     environment: str = Field(index=True)
     agents: list[str] = Field(index=True)
-    tag: str | None = Field(index=True)
-    models: list[str] | None = Field(index=True)
+    tag: str | None = Field(index=True, default="")
+    models: list[str] | None = Field(index=True, default=[])
```

Comment:
if the default is empty then is there any case it could be `None`?

Reply:
I kept that for compatibility reasons.

Reply:
I am not sure if the saved data before are none or ""/[]